src/Components/ServiceScene/Breakdowns/Panels/ValueSummary.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  PanelBuilders,
+  SceneComponentProps,
+  SceneFlexItem,
+  SceneFlexLayout,
+  sceneGraph,
+  SceneObjectBase,
+  SceneObjectState,
+  VizPanel,
+} from '@grafana/scenes';
+import { CollapsablePanelType, PanelMenu } from '../../../Panels/PanelMenu';
+import { DrawStyle, StackingMode } from '@grafana/ui';
+import { setLevelColorOverrides } from '../../../../services/panel';
+import { getPanelOption, setPanelOption } from '../../../../services/store';
+import React from 'react';
+
+const SUMMARY_PANEL_SERIES_LIMIT = 100;
+
+interface ValueSummaryPanelSceneState extends SceneObjectState {
+  body?: SceneFlexLayout;
+  title: string;
+  levelColor?: boolean;
+}
+export class ValueSummaryPanelScene extends SceneObjectBase<ValueSummaryPanelSceneState> {
+  constructor(state: ValueSummaryPanelSceneState) {
+    super(state);
+    this.addActivationHandler(this.onActivate.bind(this));
+  }
+
+  public static Component = ({ model }: SceneComponentProps<ValueSummaryPanelScene>) => {
+    const { body } = model.useState();
+    if (body) {
+      return (
+        <div>
+          <body.Component model={body} />
+        </div>
+      );
+    }
+
+    return null;
+  };
+
+  onActivate() {
+    const collapsed =
+      getPanelOption('collapsed', [CollapsablePanelType.collapsed, CollapsablePanelType.expanded]) ??
+      CollapsablePanelType.expanded;
+    const viz = buildValueSummaryPanel(this.state.title, { levelColor: this.state.levelColor });
+    const height = getValueSummaryHeight(collapsed);
+
+    this.setState({
+      body: new SceneFlexLayout({
+        key: VALUE_SUMMARY_PANEL_KEY,
+        minHeight: height,
+        height: height,
+        maxHeight: height,
+        wrap: 'nowrap',
+        children: [
+          new SceneFlexItem({
+            body: viz,
+          }),
+        ],
+      }),
+    });
+
+    this._subs.add(
+      viz.subscribeToState((newState, prevState) => {
+        if (newState.collapsed !== prevState.collapsed) {
+          const vizPanelFlexLayout = sceneGraph.getAncestor(viz, SceneFlexLayout);
+          setValueSummaryHeight(
+            vizPanelFlexLayout,
+            newState.collapsed ? CollapsablePanelType.collapsed : CollapsablePanelType.expanded
+          );
+          setPanelOption(
+            'collapsed',
+            newState.collapsed ? CollapsablePanelType.collapsed : CollapsablePanelType.expanded
+          );
+        }
+      })
+    );
+  }
+}
+
+export function setValueSummaryHeight(vizPanelFlexLayout: SceneFlexLayout, collapsableState: CollapsablePanelType) {
+  const height = getValueSummaryHeight(collapsableState);
```

Comment:
Just did a test out of curiosity and wonder why we need to manually set the height, otherwise it disappears.